contrib/depends/packages/openssl.mk

```diff
@@ -1,8 +1,8 @@
 package=openssl
-$(package)_version=1.0.2r
```

Comment:
This patch looks quite different from the original https://github.com/LMDB/bitmonero/commit/5b30203fe956990e9cf0fa7ad5a280b88b7683e8, and notably misses some configurations related to ARM building. This is why I prefer avoiding out-of-order cherry-picking.

Do you insist that this OpenSSL update is necessary now? If so, maybe it's better not to cherry-pick and make a minimal patch bumping the version only, so that the upstream changes can be picked up later.